recipes/pymodsecurity/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pymodsecurity" %}
+{% set version = "0.0.1" %}
+{% set build_number = 0 %}
+{% set hash_value = "e9570494282b71172f909cb3ecfd4b7b66fa29a3141439192a48b40939893495" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_value }}
+
+build:
+  number: {{ build_number }}
+  skip: true  # [not linux]
+  script_env:
+    - CC
+    - CXX
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - modsecurity
+    - setuptools
+    - python >=3.5
+    - pybind11 >=2.2
+    - git
+  host:
+    - modsecurity
+    - setuptools
+    - python >=3.5
+    - pybind11 >=2.2
+  run:
+    - modsecurity
+    - python >=3.5
+    - pybind11 >=2.2
+
+test:
+  imports:
+    - ModSecurity
+  commands:
+    - python -c "from ModSecurity import ModSecurity"
+    - python -c "from ModSecurity import Rules"
+
+about:
+  home: https://github.com/actions-security/pymodsecurity
+  summary: Python bindings to libmodsecurity
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers: 
+    - GustavoKatel
+    - Davysson0
```

Comment:
Could you confirm that the other person listed as a co-maintainer has agree to that role? If not, please ping them to get explicit confirmation.

Reply:
@Davysson0 can you confirm to agree be a co-mantainer role?

Reply:
Yes, I agree.